main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+    actors = (
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    )
+
+    [Genre.objects.create(name=genre) for genre in genres]
+    [Actor.objects.create(
+        first_name=first_name,
+        last_name=last_name
+    ) for first_name, last_name in actors]
```

Comment:
List comprehensions in Python are generally used for creating new lists. Using regular loops will be more clear here